uktrade/data-hub-frontend
# Add start date filters to new events collection page

## Description of change
This PR adds in 'Earliest start date' and 'Latest start date' filters to the new events collection page via modification to the Activity Stream OpenSearch query. 

The page has also been refactored so that the header and filters match the layout in the existing events collection list. 

## Test instructions
- In Django dev API, add the user feature flag user-event-activities to your adviser profile.
- Bring up this branch on your local dev frontend and navigate to http://localhost:3000/events
- You should see an 'Earliest start date' filter and a 'Latest start date' filter on the left hand side.
- It should work if you add dates in either/both these filters and should also add to query string in the url. 

## Screenshots
### Before
![image](https://user-images.githubusercontent.com/70902973/182385275-7c0c7bfe-69ea-4863-84d3-590b30bdcdf8.png)

### After
![image](https://user-images.githubusercontent.com/70902973/182385380-1b45e8ce-70e3-4a3a-af98-575561a58404.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
